src/components/Modal/SbModal.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <SbPortal
+    ref="portalRef"
+    slim
+    append
+    unmount-on-destroy
+    target-slim
+    :target="modalTarget"
+  >
+    <SbBlokUi v-if="open" @click="wrapClose">
+      <div
+        ref="modal"
+        class="sb-modal"
+        :class="{ 'sb-modal__full-width': fullWidth }"
+        role="dialog"
+        v-bind="{ ...$attrs }"
+      >
+        <button
+          class="sb-modal__close-button"
+          aria-label="Close Modal"
+          @click="handleCloseModal"
+        >
+          <SbIcon name="close" size="normal" color="primary-dark" />
+        </button>
+        <slot />
+      </div>
+    </SbBlokUi>
+  </SbPortal>
+</template>
+
+<script>
+import SbIcon from '../Icon'
+import SbBlokUi from '../BlockUI'
+import ClickOutside from '../../directives/click-outside'
+import SbPortal from '../Portal'
+import { randomString } from '../../utils'
+
+export default {
+  name: 'SbModal',
+
+  components: {
+    SbIcon,
+    SbPortal,
+    SbBlokUi,
+  },
+
+  directives: {
+    ClickOutside,
+  },
+
+  props: {
+    isOpenModal: Boolean,
```

Comment:
You can change the name of the variable to `isOpen` removing the "Modal" sufix because that's already implicit from component name.